### DISCUSSION TITLE
New Installation, Questions about bitrate and deemix

### DISCUSSION THREAD

**Discussion by mbc0** [RandomNinjaAtk/arr-scripts#62] [Q&A] [answered]
Hi,
New installation, I have 2 questions please?

1, I have audio set to lossless but I see this error in the log "2023-08-21 23:45:54 :: Audio :: 2.12 :: ERROR :: Change audioBitrate to a desired bitrate number, example: 192..."

2, How do I add deemix as an indexer?  Deemix is not an option in indexers and when searching for artists there are no deemix results.  I have entered my arl (paid deezer accont) into the extended.conf what have I missed?

Many thanks

**Comment by RandomNinjaAtk** [comment, marked as answer]
This is a script, there is no indexer. Read the config options more carefully

**Reply by mbc0** [reply]
OK, thank you, I mis-interpratated some info on discord in a convo, what about the bitrate error please?

**Reply by RandomNinjaAtk** [reply]
If you post your config and what your desired quality/output is, can give you some pointers on what to set it to. But generally, you just need to read the options carefully and set it correctly....
